MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -96,4 +100,38 @@ import Foundation
             }
         }
     }
+
+    /// Retrieves an access token for the account.
+    /// - Parameters:
+    ///   - client: The instance of Native Auth public client application.
+    ///   - forceRefresh: Ignore any existing access token in the cache and force MSAL to get a new access token from the service.
+    ///   - scopes: Optional. Permissions that should be included in the access token received after sign in flow has completed
+    ///   - correlationId: Optional. UUID to correlate this request with the server for debugging.
+    ///   - delegate: Delegate that receives callbacks for the Get Access Token flow.
+    public func getAccessToken(forceRefresh: Bool = false,
+                               scopes: [String]? = nil,
+                               correlationId: UUID? = nil,
+                               delegate: CredentialsDelegate) {
+
+        let params = MSALSilentTokenParameters(scopes: scopes ?? [], account: account)
+        params.forceRefresh = forceRefresh
+
```

Comment:
can you log something related to native auth here? So, we can understand that the next silent token retrieval is triggered by native auth.

Reply:
Do you have any idea of what to log exactly?